yt/yql/plugin/bridge/interface.h

```diff
@@ -94,6 +95,7 @@ using TFuncBridgeGetProgress = TBridgeQueryResult*(TBridgeYqlPlugin* plugin, con
     XX(BridgeFreeQueryResult) \
     XX(BridgeRun) \
     XX(BridgeGetProgress) \
-    XX(BridgeGetAbiVersion)
+    XX(BridgeGetAbiVersion) \
```

Comment:
This macro is also used here
https://github.com/ydb-platform/ydb/blob/main/yt/yql/plugin/dynamic/impl.cpp#L121

For now build is falling with the words
```
$(SOURCE_ROOT)/yt/yql/plugin/dynamic/impl.cpp:121:1: error: use of undeclared identifier 'BridgeAbort'
FOR_EACH_BRIDGE_INTERFACE_FUNCTION(XX)
^
$(SOURCE_ROOT)/yt/yql/plugin/bridge/interface.h:99:8: note: expanded from macro 'FOR_EACH_BRIDGE_INTERFACE_FUNCTION'
    XX(BridgeAbort)
       ^
1 error generated.
```

Reply:
That's sad!

For now, I've added manual calls for BridgeAbort and will remove them after YDB repository commit.